lib/projects/structure.ts

```diff
@@ -70,13 +63,20 @@ type GenericComponentConfig =
   | PrivateAppComponentConfig
   | AppCardComponentConfig;
 
+export type Component = {
+  type: ComponentTypes;
+  config: GenericComponentConfig;
+  runnable: boolean;
+  path: string;
+};
+
 export const COMPONENT_TYPES = {
   privateApp: 'private-app',
   publicApp: 'public-app',
   hublTheme: 'hubl-theme',
 } as const;
 
-type ComponentTypes = ValueOf<typeof COMPONENT_TYPES>;
+export type ComponentTypes = ValueOf<typeof COMPONENT_TYPES>;
```

Comment:
Might make sense to move these to `types/Projects`

Reply:
Good idea. I moved it all over in [086af30](https://github.com/HubSpot/hubspot-cli/pull/1335/commits/086af30e65d2a8ef88ded01a818d1388441170dd), and I made some changes that let me get rid of `COMPONENT_TYPES`